### PR TITLE
Create signup page

### PR DESCRIPTION
**Pull request by khuynh09**
Pushing the UI portion. Will fully implement cognito next! 

image

### CODE REVIEW

**Review by benjamin-huynh — APPROVED** [review]

**Comment by ChaseC99** [comment]
The box looks a little small to me but maybe that's just cuz you have a large monitor? 
@kaeleylenard, should it be wider?

File: src/components/auth/Register.js [hunk 1]
```diff
@@ -0,0 +1,411 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Container, Form, Button, Col } from "react-bootstrap";
+import MainLogo from "../../assets/raise_logo_background.png";
+import Pinwheel from "../../assets/pinwheel.png";
+import Stepper from "../standard/Stepper"
+
+const Styles = styled.div`
```
Comment (ChaseC99) [inline src/components/auth/Register.js:8]: There's inline styles all throughout the component. Please move them into here.

File: src/components/standard/Navigation.js [hunk 2]
```diff
@@ -85,7 +78,7 @@ const MainNavbar = () => {
   const Navbar = (props) => {
     return (
       <nav className="navbar">
-        <span className="logo"><img src={MainLogo} height="60px" ></img></span>
+        <span className="logo"><img alt="" src={MainLogo} height="60px" ></img></span>
```
Comment (ChaseC99) [inline src/components/standard/Navigation.js:81]: Move `height="60px" out to Styles.
Add a valid alt tag like "The Raise Foundation Logo" or something.

File: src/components/standard/Stepper.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        if (activeStep === index+1){
+            return "#0183E1"
+        }
+
+        return "#f1f1f1"
+    }
+
+    return (
+        <Styles>
+
```
Comment (ChaseC99) [inline src/components/standard/Stepper.js:26]: Nit: remove extra newline

File: src/index.css [hunk 4]
```diff
@@ -5,6 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+
```
Comment (ChaseC99) [inline src/index.css:8]: Nit: remove extra newline

File: src/components/auth/Register.js [hunk 5]
```diff
@@ -0,0 +1,411 @@
...
+                            </Form>
+                        }
+                    </div>
+                </Container>
+            </div>
+        </Styles>
+    );
+};
+
+export default Register;
```
Comment (ChaseC99) [inline src/components/auth/Register.js:411]: Like `Pages/Signup.js`, this file's indentation is 4 spaces. The rest of the project is 2 spaces. Idc which standard we go with but we should be consistent across all the files

File: src/Pages/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+const SignUp = () => {
+    return (
+        <Styles>
+            <Register />
+        </Styles>
+    );
+};
+
+export default SignUp;
```
Comment (ChaseC99) [inline src/Pages/SignUp.js:15]: This file's indentation is 4 spaces. The rest of the project is 2 spaces. Idc which standard we go with but we should be consistent across all the files.

File: src/Pages/SignUp.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (ChaseC99) [inline src/Pages/SignUp.js:1]: Nit: Let's rename this folder from `Pages` to `pages` since all of the other folders are lowercase.

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Overall looking good! Just a few things to clean up.

File: src/Pages/SignUp.js (hunk 7 above)
Comment (ChaseC99) [reply src/Pages/SignUp.js:1]: Once this is resolved, I think the changes should be good to go 👍

Comment (khuynh09) [reply src/Pages/SignUp.js:1]: hmm, its renamed on my end, i dont know why it doesn't update?

Comment (ChaseC99) [reply src/Pages/SignUp.js:1]: Oh ok, it should be fine then!

**Review by ChaseC99 — APPROVED** [review]
Good work my dude 👍

**Pull request merged**